dht_bootstrap_test.go

```diff
@@ -164,6 +164,10 @@ func TestBootstrappersReplacable(t *testing.T) {
 	}
 	require.Len(t, d.routingTable.ListPeers(), 0)
 
+	d.recentlyCheckedPeersLk.Lock()
+	d.recentlyCheckedPeers = make(map[peer.ID]time.Time)
+	d.recentlyCheckedPeersLk.Unlock()
```

Comment:
That is weird, why ? Isn't the factory function's instantiation enough ?

I guess this is to make the following `peerFound` calls predictable, then I would add a comment precising why this is done.